images.go

```diff
@@ -165,9 +168,17 @@ func (r *imageStore) Load() error {
 				}
 				names[name] = images[n]
 			}
+			// Implicit digest
 			if digest, ok := image.BigDataDigests[ImageDigestBigDataKey]; ok {
 				digests[digest] = append(digests[digest], images[n])
 			}
+			// Explicit digest
+			if image.Digest == "" {
+				image.Digest = image.BigDataDigests[ImageDigestBigDataKey]
+			}
+			if image.Digest != "" && image.Digest != image.BigDataDigests[ImageDigestBigDataKey] {
+				digests[image.Digest] = append(digests[image.Digest], images[n])
+			}
```

Comment:
Looks like these lines can be combined
```
if image.Digest == "" {
				image.Digest = image.BigDataDigests[ImageDigestBigDataKey]
} else if image.Digest != image.BigDataDigests[ImageDigestBigDataKey] {
				digests[image.Digest] = append(digests[image.Digest], images[n])
}
```

Reply:
Do we need to do anything with digests[image.Digest] if image.Digest was == ""?

Reply:
You're right, we don't.  Changing it as you've suggested.